### DISCUSSION TITLE
Original comment boxes are resized

### DISCUSSION THREAD

**Discussion by Lexxxxik** [jxlsteam/jxls#225] [Q&A] [answered]
Template has several comment boxes with some text, and size of them adjusted for it.
Using the simple implementation, I got correct result file, except the moment that original comment boxes were resized and now text are not fitted a before.

`JxlsHelper.getInstance().processTemplate(is, os, contex)`
Do I have any posibility to disable resizing? 
Original Template:
![image](https://user-images.githubusercontent.com/30318541/199774713-05631e6c-cfe9-4029-9757-38c44f0c3ad0.png)

Result:
![image](https://user-images.githubusercontent.com/30318541/199774794-1389c973-c144-49cc-b56a-c2383b2075eb.png)

**Comment by SoltauFintel** [comment, marked as answer]
Please write an enhancement issue.
PoiUtil.setCellComment() should use the sizes from the old comment or anchor.
